index.html

```diff
@@ -0,0 +1,580 @@
+            <br> If
+            true,
+            the rapid engorgement would solve paradoxes like why the heavens look uniform from pole to pole and not
+            like a jagged,
+            warped mess. The enormous ballooning would iron out all the wrinkles and irregularities. Those particles
+            were not missing,
+            but would be diluted beyond detection,
+            like spit in the ocean. <br> <br> “SPECTACULAR REALIZATION,
+            ” Dr. Guth wrote across the top of the page and drew a double box around it.
+    </article>
+    <aside>
+        <div class="editors-pick">
+                <ul class="e-picklist">
+                    <li><h3 class="picklist">Editor's Pick</h3></li>
+            <li><a href="#">
+                    <img class="illustrations" src="images/17money-alt-square640.jpg" alt="">
+                    <p>Where Are All The Wild Things,
+                        Daddy?</p>
+                </a></li>
+            <li><a href="#">
+                    <img class="illustrations" src="images/20MODERN-FATHERSDAY-square640.jpg" alt="">
+                    <p>From the Heart to Higher Education: The 2021 College <br> Essays on
+                        Money</p>
+                </a></li>
+
+            <li><a href="#">
+                    <img class="illustrations" src="images/18surfacing-pet-tube-thmpsn1-square640.jpg" alt="">
+                    <p>The Internet’s Exotic Pets, and the People Who Love Them</p>
+                </a></li>
+            </ul> </div>
+    </aside>
+
+    <article class="article-b">
+         On Monday, Dr. Guth’s starship came in. Radio astronomers reported that they had seen the beginning of
+            the
+            Big Bang, and that his hypothesis, known undramatically as inflation, looked right.
+            <br> <br>
+            Reaching back across 13.8 billion years to the <a class="cosmic-waves" href="#">first sliver of cosmic
+                time</a> with telescopes
+            at the South
+            Pole, a team of astronomers led by John M. Kovac of the Harvard-Smithsonian Center for Astrophysics
+            detected
+            ripples in the fabric of space-time — so-called <a class="cosmic-waves" href="#">gravitational waves</a>
+            —
+            the signature of a
+            universe being
+            wrenched violently apart when it was roughly a trillionth of a trillionth of a trillionth of a second
+            old.
+            They are the long-sought smoking-gun evidence of inflation, proof, Dr. Kovac and his colleagues say,
+            that
+            Dr. Guth was correct.
+            <br><br><br>
+            Inflation has been the workhorse of cosmology for 35 years, though many, including Dr. Guth, wondered
+            whether it could ever be proved.
+            <br> <br>
+            If corroborated, Dr. Kovac’s work will stand as a landmark in science comparable to the recent discovery
```

Comment:
### For your entire article section, please have a look at the original and properly position yours

![Screenshot (53)](https://user-images.githubusercontent.com/86371332/124312042-9782a200-db66-11eb-812b-ab8cda899a48.png)